apps/main/src/lend/store/createOhlcChartSlice.ts

```diff
@@ -233,29 +233,37 @@ const createOhlcChart = (set: SetState<State>, get: GetState<State>) => ({
         for (const item of ohlc) {
           const time = item.time.getLocalTimestamp()
 
-          volumeArray.push({
-            time,
-            value: item.volume,
-            color: item.open < item.close ? '#26a69982' : '#ef53507e',
-          })
+          if (item.volume && item.open && item.close) {
+            volumeArray.push({
+              time,
+              value: item.volume,
+              color: item.open < item.close ? '#26a69982' : '#ef53507e',
```

Comment:
could maybe opt for a neutral blue color if there's volume but no open and close?